docs/developer-guide/views-examples.md

```diff
@@ -0,0 +1,120 @@
+---
+description: Learn about the most used custom view examples of Lite XL.
+---
+
+## Toolbar
```

Comment:
Avoid creating consecutive heading like this. I'd prefer if you just discard the first heading and just put `Extending Views: ToolbarView`.

Reply:
What about something like the following:
![image](https://github.com/takase1121/lite-xl-docs/assets/80917080/e159432b-a94d-4b81-8182-cb6cb053bb88)

Reply:
In this case, I think we can stil leave out h2. The right TOC goes to the tutorial directly (and I think Tutorial heading should be renamed to something else, like "Creating a plugin"